src/lib/components/LeafletMap/README.md

```diff
@@ -1,3 +1,9 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// Copyright (C) 2020 - Equinor ASA.
+
```

Comment:
This one renders weirdly in Markdown. Not sure if you want it to be included in the rendered README or not?
See: https://github.com/anders-kiaer/webviz-subsurface-components/tree/change_license/src/lib/components/LeafletMap/webgl/eqGL